### PR TITLE
Add a chapter about the new pipeline syntax to the 3.0 blog post

### PR DESCRIPTION
**Pull request by Dakostu**
Add a chapter that introduces the pipelines, with links to pages that elaborate on the implementation and usage.

### CODE REVIEW

File: web/blog/vast-v3.0/index.md [hunk 1]
```diff
@@ -27,9 +25,35 @@ expressions.
...
+details on how to use the new Unix-style pipeline syntax.
+
+:::info Pipeline YAML Syntax Deprecation
+This change introduces a transitional period from YAML-style to Unix-style
+pipelines. The old YAML syntax for pipelines and for pre-v3.0 pipeline operators
+will be deprecated and removed altogether in a future version. The new pipeline
+operators [`head`][head-op] and [`taste`][taste-op] do not already contain a
+YAML syntax. The syntax of some pre-v3.0 pipeline operators will also be
+subject to change/deprecation.
+:::
```
Comment (mavam) [inline web/blog/vast-v3.0/index.md:48]:
```suggestion
## Unix-Style Pipelines

Starting with v3.0, VAST introduces a new way to write pipelines, with a syntax
similar to [splunk](https://splunk.com), [Kusto][kusto],
[PRQL](https://prql-lang.org/), and [Zed](https://zed.brimdata.io/).

[kusto]: https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/

The new syntax resembles the well-known Unix paradigm of command chaining. The
difference to Unix pipelines is that VAST exchanges structured data between
operators. The `vast export` and `vast import` commands accept such a
pipeline as a string argument. Previously, VAST only supported a YAML-like
definition of pipeline in configuration files. This new, ad-hoc form enables
much quicker data transformation.

Refer to the [pipelines documentation][pipeline-doc] for more details on how to
use the new pipeline syntax.

:::info Pipeline YAML Syntax Deprecation
This release introduces a transitional period from YAML-style to Unix-style
pipelines. The old YAML syntax for pipelines will be deprecated and removed
altogether in a future version. The new pipeline operators [`head`][head-op] and
[`taste`][taste-op] have no YAML equivalent.
:::
```

File: web/blog/vast-v3.0/index.md [hunk 2]
```diff
@@ -27,9 +25,35 @@ expressions.
...
+
+[pipeline-doc]: //vast.io/docs/understand/query-language/pipelines#define-a-pipeline
+[head-op]: //vast.io/docs/understand/query-language/operators/head
+[taste-op]: //vast.io/docs/understand/query-language/operators/taste
```
Comment (mavam) [inline web/blog/vast-v3.0/index.md:52]:
```suggestion
[pipeline-doc]: /docs/understand/query-language/pipelines#define-a-pipeline
[head-op]: /docs/understand/query-language/operators/head
[taste-op]: /docs/understand/query-language/operators/taste
```

File: web/blog/vast-v3.0/index.md [hunk 3]
```diff
@@ -27,9 +25,35 @@ expressions.
...
+
 ## VASTQL Upgrades
 
-The 3.x series of VAST releases is all about the **VAST** **Q**uery
+The 3.x series of VAST releases is also all about the **VAST** **Q**uery
```
Comment (mavam) [inline web/blog/vast-v3.0/index.md:56]: The intro now longer makes sense. This comes *after* the introduction of Unix-style pipelines.

File: web/blog/vast-v3.0/index.md [hunk 4]
```diff
@@ -27,9 +25,35 @@ expressions.
...
 **L**anguage. To kick this off, we've done some breaking changes to the language
```
Comment (mavam) [inline web/blog/vast-v3.0/index.md:57]: "To kick this off"

👆 this used to be the intro paragraph where this phrasing made sense, but needs to be reworded now.

**Review by mavam — COMMENTED** [review]
Good pitch. Wordsmithing and a few comments inline.

File: web/blog/vast-v3.0/index.md (hunk 4 above)
Comment (Dakostu) [reply web/blog/vast-v3.0/index.md]: I shortened the intro paragraph in https://github.com/tenzir/vast/pull/2921/commits/d19819131aab6ec509ef5d860e7c7346c1e56b3c

File: web/blog/vast-v3.0/index.md (hunk 3 above)
Comment (Dakostu) [reply web/blog/vast-v3.0/index.md]: I shortened the intro paragraph in https://github.com/tenzir/vast/pull/2921/commits/d19819131aab6ec509ef5d860e7c7346c1e56b3c

File: web/blog/vast-v3.0/index.md [hunk 5]
```diff
@@ -27,24 +27,27 @@ expressions.
...
 
-Refer to the ["Pipelines" section of the documentation][pipeline-doc] for more
-details on how to use the new Unix-style pipeline syntax.
+[kusto]: https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/
+
+The new syntax resembles the well-known Unix paradigm of command chaining. The
+difference to Unix pipelines is that VAST exchanges structured data between
+operators. The `vast export` and `vast import` commands accept such a
+pipeline as a string argument. Previously, VAST only supported a YAML-like
+definition of pipeline in configuration files. This new, ad-hoc form enables
```
Comment (mavam) [inline web/blog/vast-v3.0/index.md:40]:
```suggestion
definition of pipelines in configuration files. This new, ad-hoc form enables
```

File: web/blog/vast-v3.0/index.md [hunk 6]
```diff
@@ -27,10 +25,38 @@ expressions.
...
+[pipeline-doc]: /docs/understand/query-language/pipelines#define-a-pipeline
+[head-op]: /docs/understand/query-language/operators/head
+[taste-op]: /docs/understand/query-language/operators/taste
+
 ## VASTQL Upgrades
 
-The 3.x series of VAST releases is all about the **VAST** **Q**uery
-**L**anguage. To kick this off, we've done some breaking changes to the language
+We've done some breaking changes to the the **VAST** **Q**uery **L**anguage
 that we've wanted to do in quite a while. Here's a summary:
```
Comment (mavam) [inline web/blog/vast-v3.0/index.md:60]:
```suggestion
that we've wanted to do for a long time. Here's a summary:
```

**Review by mavam — APPROVED** [review]
Just a few minor nits again, otherwise LGTM.

File: web/blog/vast-v3.0/index.md (hunk 6 above)
Comment (Dakostu) [reply web/blog/vast-v3.0/index.md]: https://github.com/tenzir/vast/pull/2921/commits/84fd4d246c79883628c95a6410e3dd0bd13bd5a0

File: web/blog/vast-v3.0/index.md [hunk 7]
```diff
@@ -27,10 +25,38 @@ expressions.
...
+
+[pipeline-doc]: /docs/understand/query-language/pipelines#define-a-pipeline
+[head-op]: /docs/understand/query-language/operators/head
+[taste-op]: /docs/understand/query-language/operators/taste
+
 ## VASTQL Upgrades
 
-The 3.x series of VAST releases is all about the **VAST** **Q**uery
-**L**anguage. To kick this off, we've done some breaking changes to the language
+We've done some breaking changes to the the **VAST** **Q**uery **L**anguage
```
[the remark this answers is not in the document]
Comment (Dakostu) [reply web/blog/vast-v3.0/index.md]: https://github.com/tenzir/vast/pull/2921/commits/84fd4d246c79883628c95a6410e3dd0bd13bd5a0

**Pull request merged**